src/components/toolbar/helpers.ts

```diff
@@ -127,48 +165,36 @@ export const demoteSectionToParagraph = (
       previousNode.lastChild && isSectionNodeType(previousNode.lastChild.type)
 
     if (hasSubsections) {
-      const deepestSubsection = getDeepestSubsection(previousNode.lastChild)
-
-      const updatedDeepestSubsection = deepestSubsection.copy(
-        deepestSubsection.content
-          .append(Fragment.from(paragraph)) // Append the paragraph
-          .append(sectionContent) // Append the remaining content
-      )
-
-      const updatedPreviousNode = replaceDeepestSubsection(
```

Comment:
`replaceDeepestSubsection ` is not needed now? If yes, you can remove it.

Reply:
updated